system/Debug/Toolbar/Collectors/History.php

```diff
@@ -56,10 +58,10 @@ class History extends BaseCollector
     /**
      * Specify time limit & file count for debug history.
      *
-     * @param int $current Current history time
-     * @param int $limit   Max history files
+     * @param float $current Current history time
+     * @param int   $limit   Max history files
      */
-    public function setFiles(int $current, int $limit = 20)
+    public function setFiles(float $current, int $limit = 20)
```

Comment:
Do we need to break the signature here? I think we could just pass around the  `microtime`  as an `int` and then do `number_format` right before outputting.

Reply:
This is BC break.

Reply:
Given the limitations, I think it is acceptable to change the signature albeit a breaking change. Let's wait for other's opinions. @kenjis @MGatner ?

Reply:
I don't mind (necessary) BC in minor version up if it is documented.
See https://forum.codeigniter.com/thread-80892-post-392846.html#pid392846

By the way, is float enough?
It seems only 4 digits like 1643882644.5667 in my PHP.

Reply:
Maybe change this to a string, and make the file name look like "debugbar_{**microtime**} _ {**session_value**} .json", then we will solve the request **when multiple developers work on different modules I suggest the option that developers can filter requests by session so that everyone can see it's own requests made by their browser.**
I suggest adding a checkbox to the history tab, which would filter sessions by the browser.

Reply:
That is a good refinement to the Debug Toolbar, but let us limit this PR to adding the microtime only. We can add the session and filtering in another PR.

Reply:
I think type `string` is better. It is actually (a part of) filename, not timestamp.